<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>property_ColumnTiltMeasure</class>
 <widget class="QWidget" name="property_ColumnTiltMeasure">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>843</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Column tilt properties</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>333</width>
        <height>841</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="12" column="0" colspan="4">
        <widget class="GenericPropertiesFeet" name="genericPropsFeetWidget" native="true"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Tilt value t</string>
         </property>
        </widget>
       </item>
       <item row="13" column="0" colspan="3">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="10" column="1" colspan="3">
        <widget class="QLineEdit" name="ResultInfield">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Result reliability</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Height h</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="4">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="2">
          <widget class="QLabel" name="YLabel_2">
           <property name="text">
            <string>Y</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QDoubleEdit" name="TPYInfield">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="PointCoordLabel_2">
           <property name="text">
            <string>Bottom point</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="ZLabel_2">
           <property name="text">
            <string>Z</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QDoubleEdit" name="BPZInfield">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleEdit" name="BPXInfield">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="XLabel_2">
           <property name="text">
            <string>X</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="PointOnPlaneLabel_2">
           <property name="text">
            <string>Top point</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QDoubleEdit" name="TPZInfield">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QDoubleEdit" name="BPYInfield">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleEdit" name="TPXInfield">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0" colspan="4">
        <widget class="GenericPropertiesHead" name="genericPropsHeadWidget" native="true"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="ValuesLabel_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Coordinates</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>t/h &gt; max ratio ?</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Max ratio t/h=1/</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="3">
        <widget class="QDoubleEdit" name="TiltValuInfield">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="1" colspan="3">
        <widget class="QDoubleEdit" name="HeightInfield">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Ratio t/h=1/</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0" colspan="4">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="8" column="1" colspan="3">
        <widget class="QDoubleEdit" name="ValueMaxRatioInfield">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="4">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="2">
          <widget class="QLabel" name="YLabel">
           <property name="text">
            <string>Y</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QDoubleEdit" name="Point2YInfield">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="PointCoordLabel">
           <property name="text">
            <string>Point 1</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="ZLabel">
           <property name="text">
            <string>Z</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QDoubleEdit" name="Point1ZInfield">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleEdit" name="Point1XInfield">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="XLabel">
           <property name="text">
            <string>X</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="PointOnPlaneLabel">
           <property name="text">
            <string>Point 2</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QDoubleEdit" name="Point2ZInfield">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QDoubleEdit" name="Point1YInfield">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleEdit" name="Point2XInfield">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="9" column="1" colspan="3">
        <widget class="QLineEdit" name="OverRatioInfield">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="ValuesLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Perpendicular direction</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1" colspan="3">
        <widget class="QDoubleEdit" name="RatioInfield">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GenericPropertiesHead</class>
   <extends>QWidget</extends>
   <header location="global">GenericPropertiesHead.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GenericPropertiesFeet</class>
   <extends>QWidget</extends>
   <header>GenericPropertiesFeet.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QDoubleEdit</class>
   <extends>QLineEdit</extends>
   <header>qdoubleedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>Point1XInfield</tabstop>
  <tabstop>Point1YInfield</tabstop>
  <tabstop>Point1ZInfield</tabstop>
  <tabstop>Point2XInfield</tabstop>
  <tabstop>Point2YInfield</tabstop>
  <tabstop>Point2ZInfield</tabstop>
  <tabstop>BPXInfield</tabstop>
  <tabstop>BPYInfield</tabstop>
  <tabstop>BPZInfield</tabstop>
  <tabstop>TPXInfield</tabstop>
  <tabstop>TPYInfield</tabstop>
  <tabstop>TPZInfield</tabstop>
  <tabstop>TiltValuInfield</tabstop>
  <tabstop>HeightInfield</tabstop>
  <tabstop>RatioInfield</tabstop>
  <tabstop>ValueMaxRatioInfield</tabstop>
  <tabstop>OverRatioInfield</tabstop>
  <tabstop>ResultInfield</tabstop>
 </tabstops>
 <resources>
  <include location="../../../openscantools.qrc"/>
 </resources>
 <connections/>
</ui>
